### ISSUE TITLE
Uninitialized constant exception when using PlistHelper

### ISSUE THREAD

**Issue by vincentisambart** [CocoaPods/Xcodeproj#243]
```
$ ruby -rxcodeproj -e "Xcodeproj::PlistHelper"
/Users/vincent-isambart/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-0.22.0/lib/xcodeproj/plist_helper.rb:598:in `<module:DevToolsCore>': uninitialized constant DevToolsCore::Pathname (NameError)
	from /Users/vincent-isambart/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/xcodeproj-0.22.0/lib/xcodeproj/plist_helper.rb:542:in `<top (required)>'
	from -e:1:in `<main>'
```

It works fine if you `require "pathname"` before but that should not be needed.

**Comment by segiddins** [comment]
We'd love a PR that add the require to the plist helper!

**Comment by Fl0p** [comment]
reopen please

same error
```
[!] uninitialized constant Xcodeproj::PlistHelper
```
with xcodeproj 1.0.0.beta.3
gems list:
```
flop$ gem list

*** LOCAL GEMS ***

actionmailer (4.2.5, 4.1.0, 4.0.2)
actionpack (4.2.5, 4.1.6, 4.1.0, 4.0.2)
actionview (4.2.5, 4.1.6, 4.1.0)
activejob (4.2.5)
activemodel (4.2.5, 4.1.6, 4.0.2)
activerecord (4.2.5, 4.0.2)
activerecord-deprecated_finders (1.0.4, 1.0.3)
activesupport (4.2.5, 4.1.6, 4.1.0, 4.0.2, 3.2.16)
addressable (2.4.0)
arel (7.0.0, 6.0.3, 4.0.1)
atomic (1.1.99, 1.1.16, 1.1.14)
autoparse (0.3.3)
babelish (0.4.2)
bigdecimal (1.2.7, 1.2.0)
builder (3.2.2, 3.1.4)
bundler (1.11.2, 1.3.5)
bundler-unload (1.0.2)
bunlder (0.0.1)
CFPropertyList (2.3.2, 2.2.8)
claide (1.0.0.beta.2, 0.9.1, 0.7.0, 0.5.0, 0.4.0)
cocoapods (1.0.0.beta.5, 0.39.0, 0.32.1, 0.29.0)
cocoapods-core (1.0.0.beta.5, 0.39.0, 0.32.1, 0.29.0)
cocoapods-deintegrate (1.0.0.beta.1)
cocoapods-downloader (1.0.0.beta.1, 0.9.3, 0.5.0, 0.3.0)
cocoapods-plugins (1.0.0.beta.1, 0.4.2, 0.3.1)
cocoapods-podfile_info (1.0.0)
cocoapods-search (1.0.0.beta.1, 0.1.0)
cocoapods-stats (1.0.0.beta.3, 0.6.2)
cocoapods-trunk (1.0.0.beta.2, 0.6.4, 0.2.0)
cocoapods-try (1.0.0.beta.2, 0.5.1, 0.2.0)
cocoapods-try-release-fix (0.1.2, 0.1.1)
colored (1.2)
concurrent-ruby (1.0.0)
coveralls (0.8.10)
declarative (0.0.5)
docile (1.1.5)
domain_name (0.5.25)
erubis (2.7.0)
escape (0.0.4)
executable-hooks (1.3.2, 1.2.6)
extlib (0.9.16)
faraday (0.9.2)
fourflusher (0.3.0)
fuzzy_match (2.1.0, 2.0.4)
globalid (0.3.6)
google-api-client (0.9.1, 0.9.pre5, 0.8.6)
google_drive (1.0.5)
googleauth (0.5.1)
highline (1.7.8)
hike (2.1.3, 1.2.3)
http-cookie (1.0.2)
httpclient (2.7.1)
hurley (0.2)
i18n (0.7.0, 0.6.9)
io-console (0.4.5, 0.4.2)
json (1.8.3, 1.7.7)
json_pure (1.8.3)
jwt (1.5.2)
launchy (2.4.3)
little-plugger (1.1.4)
logging (2.0.0)
loofah (2.0.3)
mail (2.6.3, 2.5.4)
memoist (0.14.0)
mime-types (3.0, 2.99, 1.25.1)
mime-types-data (3.2015.1120)
mini_portile2 (2.1.0, 2.0.0)
minitest (5.8.3, 5.3.3, 4.3.2)
molinillo (0.4.4, 0.4.1)
multi_json (1.11.2)
multi_xml (0.5.5)
multipart-post (2.0.0)
nap (1.0.0, 0.8.0, 0.7.0)
netrc (0.11.0, 0.7.8)
nokogiri (1.6.8.rc2, 1.6.7.2)
oauth (0.4.7)
oauth2 (1.0.0)
open4 (1.3.4)
os (0.9.6)
polyglot (0.3.5, 0.3.3)
power_assert (0.2.7)
psych (2.0.17, 2.0.0)
rack (1.6.4, 1.5.2)
rack-test (0.6.3, 0.6.2)
rails (4.2.5, 4.0.2)
rails-deprecated_sanitizer (1.0.3)
rails-dom-testing (1.0.7)
rails-html-sanitizer (1.0.2)
railties (4.2.5, 4.0.2)
rake (10.5.0, 0.9.6)
rdoc (4.2.1, 4.0.0)
representable (3.0.0, 2.3.0)
rest-client (1.8.0)
retriable (2.1.0, 1.4.1)
rubygems-bundler (1.4.4, 1.4.2)
rvm (1.11.3.9, 1.11.3.8)
signet (0.7.2)
simplecov (0.11.1)
simplecov-html (0.10.0)
sprockets (3.5.2, 2.10.1)
sprockets-rails (3.0.0, 2.0.1)
term-ansicolor (1.3.2)
test-unit (3.1.7, 2.0.0.0)
thor (0.19.1, 0.18.1)
thread_safe (0.3.5, 0.1.3)
tilt (2.0.2, 1.4.1)
tins (1.6.0)
treetop (1.6.3, 1.4.15)
tzinfo (1.2.2, 1.1.0, 0.3.38)
uber (0.0.15)
unf (0.1.4)
unf_ext (0.0.7.1)
xcodeproj (1.0.0.beta.3, 0.28.2, 0.16.1, 0.14.1)
yard (0.8.7.6)
```

**Comment by segiddins** [comment]
@Fl0p please open a new issue, as this one has been closed for a long time.
